source/common/common/url_template_matcher.cc

```diff
@@ -0,0 +1,31 @@
+#include "common/common/url_template_matcher.h"
+
+#include "common/common/empty_string.h"
+#include "common/common/utility.h"
+
+namespace Envoy {
+namespace Matchers {
+
+namespace {
+
+// Http method for all url_template matches.
+// This could be anything.
+static constexpr absl::string_view HTTP_METHOD{"GET"};
```

Comment:
`static` is redundant in an anonymous namespace. clang may throw a readability warning.

Reply:
Done